projects/core/src/occ/adapters/cms/occ-cms-component.adapter.ts

```diff
@@ -57,6 +57,7 @@ export class OccCmsComponentAdapter implements CmsComponentAdapter {
       )
       .pipe(
         pluck('component'),
+        map((compts) => compts ?? []),
```

Comment:
`component` instead of `compts` would improve readability since it's what has just been plucked.  
It's a suggestion, but it's your call.